### ISSUE TITLE
Theme Dark Mode Text Contrast and Display Accessibility

### ISSUE THREAD

**Issue by brierjon** [BOINC/boinc#3112]
**Describe the bug**
The Resource share column in Advanced View and the text simple view have either white on white or Black on Gray contrast accessibility issues when using a dark theme. In Pop!_OS 19.04 enabling the default Dark Mode will reveal the screenshots. This is similar to issue #812 and #2917. Styling update is likely needed. This is using the gtk-theme https://github.com/pop-os/gtk-theme, but would guess this issue applies in all dark themes.

**Steps To Reproduce**
1. Enable Dark Mode in Pop!_OS 19.04
2. View Advanced or Simple View of BOINC Manager

**Expected behavior**
The contrast and colors would be readable and not have shade similarity.

**Screenshots**
![Screenshot from 2019-04-30 09-21-10](https://user-images.githubusercontent.com/717866/56964516-537c5880-6b29-11e9-857b-7fb684619d24.png)
![Screenshot from 2019-04-30 09-20-46](https://user-images.githubusercontent.com/717866/56964555-63943800-6b29-11e9-847a-268699f9df22.png)
![Screenshot from 2019-04-30 09-12-57](https://user-images.githubusercontent.com/717866/56964563-67c05580-6b29-11e9-9907-d30c173982f8.png)

**System Information (please complete the following information):**
 - OS: Pop!_OS 19.04 (based on Ubuntu 19.04)
 - BOINC Version: 7.14.2+dfsg-3

**Additional context**

**Comment by Ferroin** [comment]
I can confirm this on Gentoo as well using a handful of other GTK+ dark themes.  It's been the case for as long as I can remember, I've just never been bothered enough to open an issue about it.

**Comment by RichardHaselgrove** [comment]
It has been mentioned, though. We should group this with:

#812
#980
#1072
#1126
#2919
#2917

**Comment by AenBleidd** [comment]
Closed in favor of consolidated ticket #3115
